src/Explorer/Explorer.tsx

```diff
@@ -386,12 +387,16 @@ export default class Explorer {
         });
         connectionStatus.status = ConnectionStatusType.Failed;
         useNotebook.getState().resetContainerConnection(connectionStatus);
-        useDialog
-          .getState()
-          .showOkModalDialog(
-            "Connection Failed",
-            "We are unable to connect to the temporary workspace. Please try again in a few minutes. If the error persists, file a support ticket."
-          );
+        if (error?.status === HttpStatusCodes.Forbidden && error.message) {
```

Comment:
Did you verify this by changing the BRS value for max dbaccounts per user? The error you throw wont have a status right? this wont work based on the current check